src/burnchains/burnchain.rs

```diff
@@ -1213,12 +1213,21 @@ impl Burnchain {
         );
 
         if let Some(target_block_height) = target_block_height_opt {
-            if target_block_height < end_block {
+            // target_block_height is used as a hint, but could also be completely off
+            // in certain situations. The current function is directly reading the
```

Comment:
s/"The current function"/"This function"

Reply:
Note: you can actually propose some changes using the suggestion syntax - https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request

Reply:
```suggestion
            // in certain situations. This function is directly reading the
```